### PR TITLE
Project Management Basics

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**

### CODE REVIEW

**Comment by michaelmattig** [comment]
I noticed
 - A new default project is created at each login (of a registered user) instead of only the first login
 - The "Save As" feature is changed. In Mapping it created a copy, now it renames. It's fine for me to support only rename for now but please call it "Rename". It doesn't match users expectation of "Save as".
 - I had some 400 errors on project update that I unfortunately could not reproduce. Nothing to do here now I guess but keep an eye out for it.

**Comment by ChristianBeilschmidt** [comment]
> I noticed
> 
>     * A new default project is created at each login (of a registered user) instead of only the first login
> 
>     * The "Save As" feature is changed. In Mapping it created a copy, now it renames. It's fine for me to support only rename for now but please call it "Rename". It doesn't match users expectation of "Save as".
> 
>     * I had some 400 errors on project update that I unfortunately could not reproduce. Nothing to do here now I guess but keep an eye out for it.

grafik

You get the three requests:

1. `/login` --> field `project` is empty
2. `/project`--> creates a new project
3. `/session/project/2b27232e-5812-48a3-be7b-ebf306e3b7cf` --> sets the new project at the backend to the user

But when you login again, `project` is still empty

**Comment by michaelmattig** [comment]
> You get the three requests:
    /login --> field project is empty
    /project--> creates a new project
    /session/project/2b27232e-5812-48a3-be7b-ebf306e3b7cf --> sets the new project at the backend to the user
>
>But when you login again, project is still empty

That's because when you login again you get a new session where no project is yet selected. The selected project is attached to the session, not to the user. This way a user can have multiple sessions with different projects. The question is, which project should be loaded for a new session. Creating a new one each time would only make sense if the user actively decides to keep/discard projects. Otherwise you end up with hundreds of projects. I would propose to load the most recent project. So on login, request the list of projects, order DateDec, offset 0, limit 1. Then set the project as session project. If the list is empty, create a default project. What do you think?

File: projects/wave-core-new/src/lib/backend/backend.service.ts [hunk 1]
```diff
@@ -63,6 +72,7 @@ export class BackendService {
             bounds: STRectangleDict,
         },
         sessionId: UUID): Observable<CreateProjectResponseDict> {
+        console.log('create', request); // TODO: remove log
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/backend/backend.service.ts:75]:
```suggestion

```

**Pull request merged**